index.html

```diff
@@ -68,7 +68,340 @@ <h3>
+              <li class="projectBootstrap">
+                  <ul>
+                      <li>bootstrap</li>
+                  </ul>
+              </li>
+              <li class="projectRuby">
+                  <ul>
+                      <li>ruby</li>
+                  </ul>
+              </li>
+          </ul>
+      </div>
+      <div class="projectButton">
+          <button type="button">See Project</button>
+      </div>
+  </div>
+
+  <div class="project">
+      <div class="projectHeader">
+          <h2>Profesional Art Printing Data</h2>
+      </div>
+
+      <div class="projectBody">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul>
+              <li class="projectHTML">
+                  <ul>
+                      <li>html</li>
+                  </ul>
+              </li>
+              <li class="projectBootstrap">
+                  <ul>
+                      <li>bootstrap</li>
+                  </ul>
+              </li>
+              <li class="projectRuby">
+                  <ul>
+                      <li>ruby</li>
+                  </ul>
+              </li>
+          </ul>
+      </div>
+      <div class="projectButton">
+          <button type="button">See Project</button>
+      </div>
+  </div>
+</section>
+
+</section>
+</main>
+<section class="about" id="about-section">
+  <div class="about-details">
+      <div class="abt-container">
+
+  
+          <div class="abt">
+       
+      <div class="d-abt">
+
+      <div class="abt-contain">
+
+    
+           <h2>
+              About me
+           </h2>
+        
+           <div class="abt-details">
+           <p>
+              Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+             
+           </p> 
+           </div>
+           <section class="abt-btn">
+              <button class="get-btn">Get My Resume</button>
+          </section>  
+           </div>
+          </div>
+       </div>
+       <hr class="bg"> 
+      </div>
+       </div>
+  <section class="about-details">
+<ul class="technologies">
+  <li><h4><span class="kite"></span>Language </h4> 
+      <ul>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+      </ul>
+  </li>
+  <li><h4><span class="sqr"></span>Frameworks</h4> 
+      <ul>
+          <li>Bootstrap</li>
+          <li>Ruby on Rails</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+      </ul>
+  </li>
+  <li class="m-b" ><h4><span class="circle"></span>Skills </h4>
+      <ul>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+      </ul>
+  </li>
+</ul>
```

Comment:
- [ ] Please note that the technologies list's titles and content should be aligned in center & the text inside of them should be aligned center as well to match figma design


Figma           | Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/89282221/192111408-0e14f03e-318b-4a88-94f0-5c028bba4fbe.png) | ![image](https://user-images.githubusercontent.com/89282221/192111356-45a94c5b-1689-49a9-be80-42cc3b14eef6.png)